onnxscript/version_converter/version_converter_test.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+from __future__ import annotations
+
+import unittest
+
+import onnx.checker
+import onnx.parser
+import onnx.shape_inference
+
+from onnxscript import ir
+from onnxscript.version_converter import convert_version
+
+
+class VersionConverter18to19Test(unittest.TestCase):
+    def test_version_convert_compatible(self):
+        model_proto = onnx.parser.parse_model(
+            """
+            <ir_version: 7, opset_import: [ "" : 18]>
+            agraph (float[1, 4, 512, 512] input_x, float[1, 4, 512, 64] input_y) => (float[1, 4, 512, 64] output)
+            {
+                shape_a = Constant<value: tensor = int64[3] {4, 512, 512}>()
+                reshape_x = Reshape (input_x, shape_a)
+                shape_b = Constant<value: tensor = int64[3] {4, 512, 64}>()
+                reshape_y = Reshape (input_y, shape_b)
+                matmul = MatMul (reshape_x, reshape_y)
+                shape_c = Constant<value: tensor = int64[4] {1, 4, 512, 64}>()
+                output = Reshape (matmul, shape_c)
+            }
+        """
+        )
+        model = ir.serde.deserialize_model(model_proto)
+        target_version = 19
+        new_model = convert_version(model, target_version=target_version)
```

Comment:
Where is `convert_version` defined? I couldn`t find it. Anyway: I don't think it should return a model, since it will update the input model destructively.